packages/app/src/App.tsx

```diff
@@ -183,6 +186,7 @@ const routes = (
       {techDocsPage}
       <TechDocsAddons>
         <ReportIssue />
+        <TextSize />
```

Comment:
Should I add it to the embedded app in the TechDocs CLI?

Reply:
Hmm... Either way!

Would be nice to figure out a way for CLI users to pick which are enabled/disabled... But that's a rabbit hole for another time.

Reply:
Agree, thanks!